diff --git a/node_modules/cordova-plugin-background-mode/src/android/ForegroundService.java b/node_modules/cordova-plugin-background-mode/src/android/ForegroundService.java
index 806c956..eb19e18 100644
--- a/node_modules/cordova-plugin-background-mode/src/android/ForegroundService.java
+++ b/node_modules/cordova-plugin-background-mode/src/android/ForegroundService.java
@@ -222,7 +222,7 @@ public class ForegroundService extends Service {
             intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP | Intent.FLAG_ACTIVITY_SINGLE_TOP);
             PendingIntent contentIntent = PendingIntent.getActivity(
                     context, NOTIFICATION_ID, intent,
-                    PendingIntent.FLAG_UPDATE_CURRENT);
+                    PendingIntent.FLAG_UPDATE_CURRENT | PendingIntent.FLAG_IMMUTABLE);
 
 
             notification.setContentIntent(contentIntent);
